docs/cub/developer_overview.rst

```diff
@@ -635,13 +646,74 @@ Finally, the tuning policy hub looks like:
       using MaxPolicy = Policy600; // alias where policy selection is started by ChainedPolicy
     };
 
-The tuning (hub) consists of a class template, possibly parameterized by tuning-relevant compile-time parameters,
-containing a list of policies.
-These policies are chained by inheriting from ChainedPolicy
+The policy hub is a class template, possibly parameterized by tuning-relevant compile-time parameters,
+containing a list of policies, one per minimum PTX version (i.e., SM architecture) they target.
+These policies are chained by inheriting from ``ChainedPolicy``
 and passing the minimum PTX version where they should be used,
-as well as their own policy type and next lower policy type.
+as well as their own policy type and the next lower policy type.
 An alias ``MaxPolicy`` serves as entry point into the chain of tuning policies.
-Each policy then defines sub policies for each agent, since a CUB algorithm may use multiple kernels/agents.
+
+
+Tunings
+====================================
+
+Because the values to parameterize an agent may vary a lot for different compile-time parameters,
+the selection of values can be further delegated to tunings.
+Often, such tunings are found by experimentation or heuristic search.
+See also :ref:`cub-tuning`.
+
+Tunings are usually organized as a class template, one per PTX version,
+with a template specialization for each combination of the compile-time parameters,
+for which better values for an agent policies are known.
+An example set of tunings could look like this:
+
+.. code-block:: c++
+
+    template <int ValueSize, bool IsPlus>
+    struct sm60_tuning { // default tuning
+        static constexpr int threads = 128;
+        static constexpr int items = 16;
+    };
+
+    template <>
+    struct sm60_tuning<4, true> { // tuning for summing 4-byte values
+        static constexpr int threads = 256;
+        static constexpr int items = 20;
+    };
+
+    template <int ValueSize>
+    struct sm60_tuning<ValueSize, true> { // tuning for summing values of other sizes
```

Comment:
I love examples and these three blocks show a fantastic example.

Reply:
Thx!